study-builder/tenants/testboston/pages/login.html

```diff
@@ -84,37 +84,66 @@
     var headerText = "";
     var studyColor = '#2760b2';   // button primary color
     var studyHelpEmail = '##STUDY_EMAIL##';
-
-    var title = 'Welcome!';
-    if (mode === 'login') {
-        title = 'Please Sign In';
-    }
-    else if (mode === 'signup') {
-        title = 'Create your account';
-    }
-
     var loginUrl = config.callbackURL + '/login';
     var registerUrl = '##BASE_URL##';
-
     var connection = config.connection;
     var prompt = !prompt;
-    var languageDictionary;
-    var language;
     var showPasswordRecentText = false;
     var doNotShowRegLink = false;
+    var language = config.extraParams.language || 'en';
 
-    languageDictionary = { title: title,signupTitle: title};
+    var i18n = {};
+    i18n['en'] = {
+      error: {
+        signUp: {
+          userExists: 'If you have already registered, sign in here.'
+        }
+      },
+      fields: {
+        firstNamePlaceholder: "Your first name",
+        lastNamePlaceholder: "Your last name",
+        nonBlankHint: "Can't be blank"
+      },
+      lastLoginInstructions: 'Last time you signed in with',
+      message: {
+        alreadyRegisteredText: 'If you have already registered, sign in here.',
+        helpHtml: 'For help, please contact ' + studyHelpEmail + '.',
```

Comment:
This one is concatenated with other HTML and put into `innerHTML` so I named it `helpHtml`.